.changeset/angry-spies-allow.md

```diff
@@ -0,0 +1,6 @@
+---
+"stylelint": patch
+---
+
+Fixed: `keyframe-selector-notation` false positives for `entry-crossing` and `exit-crossing`
+Fixed: `keyframe-selector-notation` false negatives for `enter`
```

Comment:
Can we do two entries?
I vaguely remember that changeset has an issue with this.

Reply:
I tested `parse.default` using https://npm.runkit.com/%40changesets%2Fparse; it worked as intended.
i.e. produces `{releases: [{name: "stylelint", type: "patch"}], summary: "Fixed: foo\nFixed: bar"}`

Reply:
@romainmenke check https://github.com/stylelint/stylelint/pull/7867#discussion_r1686023873